electroncash/commands.py

```diff
@@ -547,20 +555,20 @@ def _mktx(self, outputs, fee=None, change_addr=None, domain=None, nocheck=False,
         return tx
 
     @command('wp')
-    def payto(self, destination, amount, fee=None, from_addr=None, change_addr=None, nocheck=False, unsigned=False, password=None, locktime=None,
+    def payto(self, destination, amount, fee=None, feerate=None, from_addr=None, change_addr=None, nocheck=False, unsigned=False, password=None, locktime=None,
```

Comment:
I'm really stupid -- I forgot how this works.  Nothing calls these with pure positional args, right? They are 100% always called by the system with named args?  In other words -- it's safe to add args here?

Sorry I just spent to little time on the `commands.py` subsystem...

Reply:
Not what I meant. Let's say you call command
electron-cash payto x y z --arg test

it should call payto("x","y","z", arg="test")
Basically the magic here is that it parses function signature, and all params which have no default are args, those which have defaults are kwargs. And looks like it isn't even possible to provide fee positionally for example, not fully sure. And what I also meant is, in most cases, even if it is still possible, it doesn't make sense to provide most args positionally, like payto(address, amount, None, from_addr), people probably call payto(address, amount, from_addr=from_addr)

Reply:
If it will make you feel safer we can of course move it to the end, just that it once again makes sense to have feerate go after fee (:

Reply:
I do like it near fee and not at the end.

Well if you say it’s safe .. I believe you. At this point you are more an expert on the CLI subsystem than I am and I feel comfortable with your expertise here.

also if it breaks you’ll likely fix it so.. sounds good to me.